Classes/Repository/RepositoryIssuesViewController.swift

```diff
@@ -94,6 +99,19 @@ IndicatorInfoProvider {
             }
         }
     }
+    func composeIndicatorTitle(with itemCount: Int) -> String? {
+        var newTitle: String
+        switch type {
+        case .issues: newTitle = "Issues (\(itemCount))"
+        case .pullRequests: newTitle = "Pull Requests (\(itemCount))"
+        }
+        guard itemCount > 0 else { return nil }
```

Comment:
I think it makes more sense for to `guard` agains `itemCount` in top of the function body.